space2stats_api/src/tests/test_api.py

```diff
@@ -24,15 +23,43 @@
     "properties": {},
 }
 
+@pytest.fixture(scope="session")
+def database(postgresql_proc):
+    """Fake Database."""
+    with DatabaseJanitor(
+        user=postgresql_proc.user,
+        host=postgresql_proc.host,
+        port=postgresql_proc.port,
+        dbname="testdb",
+        version=postgresql_proc.version,
+        password="password",
+    ) as jan:
+        yield jan
```

Comment:
ideally we would add data to this database fixture and run the API against it instead of `mocking` the response

Reply:
@vincentsarago; do you want us to add this within the scope of this PR or should we make a separate PR afterwards?

Reply:
we'll make a separate PR for this